src/core/interfaces.d.ts

```diff
@@ -124,6 +124,23 @@ export interface Theme {
 	[key: string]: object;
 }
 
+export interface Variant {
+	root: string;
+}
+
+export interface ThemeVariant {
+	theme: Theme | ThemeVariantConfig;
```

Comment:
Do you think `theme` should now be called `css`? Would make it less weird when accessing them, so would be `theme.css` over `theme.theme`